info/v1/container.go

```diff
@@ -922,13 +922,23 @@ type ProcessStats struct {
 	Ulimits []UlimitSpec `json:"ulimits,omitempty"`
 }
 
+type PerfError struct {
+	EventName string `json:"event_name"`
+	// Action which caused raising an error i.e. opening perf event
+	Action    string `json:"action"`
+	ErrorCode int    `json:"error_code"`
```

Comment:
please add comment to all elements in `PerfError` to keep the same style as it is in `Perf` struct.